app/javascript/stylesheets/pages/singleClass.scss

```diff
@@ -0,0 +1,209 @@
+.path-wrapper {
+  background-color: $shadow-color;
+  display: flex;
+  padding-left: 1rem;
+  line-height: 1rem;
+}
+
+.path {
+  color: $dark-color;
+  text-decoration: none;
+  cursor: pointer;
+  font-size: 0.75rem;
+  padding: 0.5rem 0;
+}
+
+.class-header {
+  background-color: var(--colorCategory);
+  color: $white-color;
+  padding: 1.25rem 0 1.25rem 1rem;
+
+  &-title {
```

Comment:
**issue (blocking)**: it's not a good practice to use sass to generate class names, if in the future you change the name of "class-header-title" and do a global search this will not appear in the results. Also, this should not be inside "class-header" as it's not a modifier or state.

Reply:
I believe the search doesn't have to be done for the whole term, and if you know the parent of the nested term, that's all you need. Yesterday I was advised by Zé to do this in a way to clear and organize my code and prevent repetitions. Despite this, if you want I can redo it.